src/entry.c

```diff
@@ -976,6 +976,52 @@ vstumpless_set_entry_message( struct stumpless_entry *entry,
   return entry;
 }
 
+const char *
+stumpless_get_entry_procid( const struct stumpless_entry *entry ) {
+  char *procid;
+  
+  VALIDATE_ARG_NOT_NULL( entry );
+
+  procid = alloc_mem( sizeof(char) * 129 );
```

Comment:
Use the `STUMPLESS_MAX_PROCID_LENGTH+1` expression here instead of a magic number. This is more readable, and most compilers will still convert this to a constant expression.